### PR TITLE
Client Render Entity Model mappings

### PR DESCRIPTION
**Pull request by jmendonza**
Some mappings to the Client.Render.Entity.Model ...

### CODE REVIEW

File: mappings/net/minecraft/client/render/entity/model/BannerBlockEntityModel.mapping [hunk 1]
```diff
@@ -1 +1,7 @@
 CLASS net/minecraft/class_550 net/minecraft/client/render/entity/model/BannerBlockEntityModel
+	FIELD field_3309 banner Lnet/minecraft/class_630;
+	FIELD field_3310 stickHorizontal Lnet/minecraft/class_630;
+	FIELD field_3311 stickVertical Lnet/minecraft/class_630;
```
Comment (Runemoro) [inline mappings/net/minecraft/client/render/entity/model/BannerBlockEntityModel.mapping]: Yarn names should be in the usual word order: `horizontalStick`, `verticalStick`.

File: mappings/net/minecraft/client/render/entity/model/BatEntityModel.mapping [hunk 2]
```diff
@@ -1 +1,7 @@
 CLASS net/minecraft/class_553 net/minecraft/client/render/entity/model/BatEntityModel
+	FIELD field_3319 wingRightTip Lnet/minecraft/class_630;
+	FIELD field_3320 wingLeft Lnet/minecraft/class_630;
+	FIELD field_3321 head Lnet/minecraft/class_630;
+	FIELD field_3322 wingRight Lnet/minecraft/class_630;
+	FIELD field_3323 body Lnet/minecraft/class_630;
+	FIELD field_3324 wingLeftTip Lnet/minecraft/class_630;
```
Comment (Runemoro) [inline mappings/net/minecraft/client/render/entity/model/BatEntityModel.mapping]: `rightWingTip`, `leftWing`, `rightWing`, `leftWingTip`

**Comment by Runemoro** [comment]
All of the names are in the wrong word order, see [the conventions](https://github.com/FabricMC/yarn/blob/19w34a/CONVENTIONS.md#general).

File: mappings/net/minecraft/client/render/entity/model/BatEntityModel.mapping (hunk 2 above)
Comment (jmendonza) [reply mappings/net/minecraft/client/render/entity/model/BatEntityModel.mapping]: https://github.com/FabricMC/yarn/pull/825/commits/c63132b3b88c2d9d95b23fae485b061d82b69c44

File: mappings/net/minecraft/client/render/entity/model/BannerBlockEntityModel.mapping (hunk 1 above)
Comment (jmendonza) [reply mappings/net/minecraft/client/render/entity/model/BannerBlockEntityModel.mapping]: https://github.com/FabricMC/yarn/pull/825/commits/c63132b3b88c2d9d95b23fae485b061d82b69c44

File: mappings/net/minecraft/client/render/entity/model/ChickenEntityModel.mapping [hunk 3]
```diff
@@ -1,2 +1,9 @@
 CLASS net/minecraft/class_558 net/minecraft/client/render/entity/model/ChickenEntityModel
+	FIELD field_3340 beak Lnet/minecraft/class_630;
+	FIELD field_3341 leftWing Lnet/minecraft/class_630;
+	FIELD field_3342 wattle Lnet/minecraft/class_630;
+	FIELD field_3343 RightLeg Lnet/minecraft/class_630;
```
Comment (Runemoro) [inline mappings/net/minecraft/client/render/entity/model/ChickenEntityModel.mapping]: `rightLeg`

Comment (jmendonza) [reply mappings/net/minecraft/client/render/entity/model/ChickenEntityModel.mapping]: https://github.com/FabricMC/yarn/pull/825/commits/ba7ca63d44482548d4b16680129e93626b1769c9

**Review by Runemoro — APPROVED** [review]

File: mappings/net/minecraft/client/render/entity/model/BatEntityModel.mapping [hunk 4]
```diff
@@ -1 +1,7 @@
 CLASS net/minecraft/class_553 net/minecraft/client/render/entity/model/BatEntityModel
+	FIELD field_3319 rightWingTip Lnet/minecraft/class_630;
+	FIELD field_3320 leftWing Lnet/minecraft/class_630;
+	FIELD field_3321 head Lnet/minecraft/class_630;
+	FIELD field_3322 righWing Lnet/minecraft/class_630;
```
Comment (modmuss50) [inline mappings/net/minecraft/client/render/entity/model/BatEntityModel.mapping]: `rightWing`

**Review by modmuss50 — CHANGES_REQUESTED** [review]
One typo, other than that looks good.

File: mappings/net/minecraft/client/render/entity/model/BatEntityModel.mapping (hunk 4 above)
Comment (jmendonza) [reply mappings/net/minecraft/client/render/entity/model/BatEntityModel.mapping]: https://github.com/FabricMC/yarn/pull/825/commits/ce2f24233b4d6475c4d82f9c0dfc21c517cda6d7

**Review by modmuss50 — APPROVED** [review]

**Comment by modmuss50** [comment]
Squished and merged via my IDE, thanks.

**Pull request closed without merging**